pkg/cloudprovider/vsphereparavirtual/cloud.go

```diff
@@ -140,7 +140,6 @@ func (cp *VSphereParavirtual) Instances() (cloudprovider.Instances, bool) {
 // InstancesV2 returns an implementation of cloudprovider.InstancesV2.
 //  TODO: implement this for v1.20
 func (cp *VSphereParavirtual) InstancesV2() (cloudprovider.InstancesV2, bool) {
-	klog.Warning("The vSphere cloud provider does not support InstancesV2")
```

Comment:
actually not so sure if we can remove this - but it's a little noisy every time we debug something

Reply:
I think it's safe, thanks for doing that